poisson_lib.py

```
@@ -186,12 +204,14 @@ def check_count_outcome(series: pd.Series) -> Tuple[bool, str]:
 def analyze_poisson_outcome(
-    outcome_name: str, 
-    df: pd.DataFrame, 
-    var_meta: Optional[Dict[str, Any]] = None, 
-    offset_col: Optional[str] = None, 
-    interaction_pairs: Optional[List[Tuple[str, str]]] = None
-) -> Tuple[str, Dict[str, Any], Dict[str, Any], Dict[str, Any]]:
+    outcome_name: str,
+    df: pd.DataFrame,
+    var_meta: Optional[Dict[str, Any]] = None,
+    offset_col: Optional[str] = None,
+    exclude_cols: Optional[List[str]] = None,
```

Ruff (0.14.13)

Unused function argument: `exclude_cols`

(ARG001)